src/main/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionContext.java

```diff
@@ -58,6 +58,13 @@
     private final Map<Long, VP8FrameProjection>
         vp8FrameProjectionMap = new ConcurrentHashMap<>();
 
+    /**
+     * A map that stores the maximum sequence number of frames that are not
+     * (yet) accepted/projected.
```

Comment:
can you add something about the key, value of the map? i.e. `Maps ssrc to sequence number` or whatever it is.